OpenEdXMobile/res/layout/edx_error_state.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools">
+
+    <FrameLayout
```

Comment:
use case to add `FrameLayout`?

Reply:
layout margin wasn't working when included in another layout.